campaigns/inferno_deal_with_the_devil.tex

```diff
@@ -40,7 +40,8 @@ \subsection*{\MakeUppercase{Player Setup}}
 
 \textbf{Town Buildings:} \svgunit{bronze} Dwelling, \svgunit{silver} Dwelling, City Hall, Citadel
 
-\textbf{Bonus:} Add a ``war machine'' of your choice to your hand.
+\textbf{Bonus:} Add a War Machine Card of your choice to your hand.
+\vspace*{\fill}\columnbreak
```

Comment:
Another good spot for some art. This time, let's go with Vision. The same rules apply s for the Land Mine.

![vision](https://github.com/user-attachments/assets/8a45d9d2-a55b-4d8c-8261-27615ecf0677)